### ISSUE TITLE
confused about documentation's data system upward/downward data flow

### ISSUE THREAD

**Issue by lingtalfi** [Polymer/docs#1731]
In this section: 
https://www.polymer-project.org/1.0/docs/devguide/data-system#upward-and-downward-data-flow

Below the image, there is this sentence: "When a property changes on the host element, it sets the corresponding property on the target element, triggering the associated property effects."

Could you prove it with code please?

**Comment by arthurevans** [comment]
@lingtalfi I'm sorry, I don't understand what you mean by "prove it with code". Are you looking for code examples like the ones that immediately precede this section?

Thanks,
Arthur

**Comment by lingtalfi** [comment]
@arthurevans yes, that's what I meant. The reason for this is, I believe, there is a lot of nomenclature that is taken for granted (words like host, property, target, property effects...). 

As a newbee, my first reflex was to search for a (centralized) nomenclature section in the docs,
but didn't found it, so what's left is read the whole documentation pages a few times and GUESS (tedious) what was meant.

In that case where understanding is not trivial (I'm foreign), examples are the only things I can really rattach to, because I can test them, and then have the necessary aha moment that keeps me going. 

So to rephrase, what I meant was really: please add more examples, and make the examples more complete (i.e. don't assume that the reader know the polymer concepts).
